azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/redux/action/ParticipantAction.kt

```diff
@@ -18,4 +18,5 @@ internal sealed class ParticipantAction : Action {
         ParticipantAction()
 
     class ReadReceiptReceived(val infoModel: ParticipantTimestampInfoModel) : ParticipantAction()
+    class ParticipantToHideReceived(val id: String) : ParticipantAction()
```

Comment:
We know for GA release, we will have a API that allow Contosos to pass a list of masked users. I think we should make this action `ParticipantsToHideReceived` and takes a list of ids already.